docs/builder/manifest.md

```diff
@@ -0,0 +1,287 @@
+# Customizing Your Web Manifest
+
+Many capabilities of progressive web apps are exposed through your app's web manifest, and you can use the PWABuilder service to edit your manifest and enhance your app.
+
+
+To edit your manifest via the PWABuilder site:
+
+1. Get to your app's report card page. If you need guidance on this, vist the [Quick Start.](/builder/quick-start)
+
+2. In the `Manifest` subsection, click `Edit Your Manifest`.
+
+3. A modal with several tabs will open. Each tab will contain different settings for editing your manifest, along with brief tooltips on what each field means.
+
+4. Once you've edited your manifest to your liking, you can either download your manifest with the `Download Manifest` button at the bottom of the modal, or copy-paste it from the `Code` tab on the far right.
+
+If you want more in depth documentation on manifest fields, full descriptions for each field follow below. They are separated into the sections they appear in on the PWABuilder website.
+
+## Info
+
+### name: `string`
+
+`name` is a required field that represents the display name for your application. Anywhere where a name for this application would be displayed, this value will be used. 
+
+This name should usually align with the store listings associate with your applications.
+
+The name of your application also must be at least two characters in length.
+
+```json
+"name": "WebBoard: A Drawing App"
+```
+
+### short_name: `string`
+
+`short_name` functions similarly as the `name` field, except that it will only be used when there is not enough character space to display the applications regular name. 
+
+It is recommended that `short_name` be 12 characters or less in length.
+
+```json
+"short_name": "WebBoard"
+```
+
+?> `short_name` isn't required by Web Standards, but is a required field for packaging with the PWABuilder service.
+
+### description: `string`
+
+`description` is an optional field that can be used to describe the functionality and purpose of your app.
+
+Just like `short_name`, this data should usually align with any store listings.
+
+```json
+"description": "WebBoard is your go to application for quick doodles, notes, or sketches!"
+```
+
+### background_color: `string`
+
+`background_color` is an optional field that represents the page color of the window that your application will be opened in. This is the color that your app will default to before any styles are loaded. Once styles are loaded, your application will use the background color defined in your CSS.
+
+```json
+"background_color": "green"
```

Comment:
background_color and theme_color need to be hex values on PWABuilder so I think that you should put hex values here. You can put a note similar to short name about the difference there but yeah